bukkit/src/main/java/com/viaversion/viaversion/bukkit/providers/BukkitPickItemProvider.java

```diff
@@ -155,7 +164,7 @@ private void spawnItem(final ItemStack item, final PlayerInventory inventory, fi
         }
 
         inventory.setItem(emptySlot, heldItem);
-        inventory.setItemInMainHand(item);
+        inventory.setItem(EquipmentSlot.HAND, item);
```

Comment:
This definitely doesn't exist in 1.8

Reply:
it does in sportpaper, unsure about spigot

Reply:
Spigot 1.8.8 only have `setItemInHand`, `setItemInMainHand` alse doesn't exists

Reply:
Yeah i was aware of setItemInMainHand not existing, its why i changed it, will change it further